### PR TITLE
Update zh-cn translation

### PR DESCRIPTION
**Pull request by LeonXu98**

### CODE REVIEW

File: src/main/resources/assets/multiconnect/lang/zh_cn.json [hunk 1]
```diff
@@ -1,3 +1,9 @@
 {
-  "multiconnect.changeForcedProtocol": "版本"
+  "multiconnect.betaWarning.line1": "当前 %s 的支持在multiconnect属于测试版功能。",
```
Comment (Earthcomputer) [inline src/main/resources/assets/multiconnect/lang/zh_cn.json:2]: Are there meant to be spaces either side of the %s?

File: src/main/resources/assets/multiconnect/lang/zh_cn.json [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  "multiconnect.betaWarning.line2": "可能会出现问题，务必向问题追踪器汇报！",
+  "multiconnect.changeForcedProtocol": "版本",
+  "multiconnect.rareBug": "multiconnect发生了一个不常见的问题。请上传你的游戏日志到 %s 以确保它能被修复。",
```
Comment (Earthcomputer) [inline src/main/resources/assets/multiconnect/lang/zh_cn.json:5]: same here

Comment (LeonXu98) [reply src/main/resources/assets/multiconnect/lang/zh_cn.json:5]: These spaces added by me mistakly. Just remove these.

**Review by Earthcomputer — APPROVED** [review]
Ty

**Pull request merged**